src/test/java/ControllerTests.java

```diff
@@ -1,20 +1,84 @@
 import controller.Controller;
+import model.Criteria;
+import model.Rubric;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
-import static org.junit.jupiter.api.Assertions.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
+import static org.junit.jupiter.api.Assertions.*;
 
 
 public class ControllerTests {
-    private final Controller controller = new Controller();
+
+    private Controller controller;
+    private List<Criteria> criteriaList;
+
+
+    @BeforeEach
+    public void init() throws Exception {
+        controller = new Controller();
+        criteriaList = new ArrayList<>(Arrays.asList(
+                new Criteria("Criteria Unu"),
+                new Criteria("Criteria Doi"),
+                new Criteria("Criteria Trei"),
+                new Criteria("Criteria Patru"),
+                new Criteria("Criteria Cinci"),
+                new Criteria("Criteria Sase")));
+    }
+
 
     @Test
     public void testAddRubricWithNullTitle() {
-        assertNull(controller.addRubric(null));
+
+        Exception exception = assertThrows(Exception.class, () -> controller.addRubric(new Rubric(null, new ArrayList<>())));
+
+        String expectedMessage = "Rubric Title cannot be null!";
+        String actualMessage = exception.getMessage();
+
+        assertEquals(expectedMessage, actualMessage);
     }
 
     @Test
-    public void testAddRubricWithValidTitle() {
-        assertEquals("Sample Title", controller.addRubric("Sample Title").getTitle());
+    public void testAddRubricWithValidTitle() throws Exception {
```

Comment:
These changes should have been done in the previous task as they are unrelated to this task. The changes have been made for better error handling that was not implemented before.